home/tests/test_serializers.py

```diff
@@ -0,0 +1,258 @@
+import pytest
+
+from home.models import ContentPage, HomePage
+from home.serializers import (
+    format_whatsapp_message,
+    has_next_message,
+    has_previous_message,
+)
+
+from .page_builder import (
+    MBlk,
+    MBody,
+    PageBuilder,
+    SBlk,
+    SBody,
+    UBlk,
+    UBody,
+    VarMsg,
+    VBlk,
+    VBody,
+    WABlk,
+    WABody,
+)
+
+
+def create_content_page(
+    wa_body_count: int = 1,
+    wa_var: int = 0,
+) -> ContentPage:
+    """
+    Helper function to create pages needed for each test.
+
+    Parameters
+    ----------
+    wa_body_count : int
+        How many WhatsApp message bodies to create on the content page.
+    wa_var: int
+        How many WhatsApp variation message bodies to create on the content page. All variations will be on gender
+    """
+    title = "default page"
+    home_page = HomePage.objects.first()
+    main_menu = PageBuilder.build_cpi(home_page, "main-menu", "Main Menu")
+    parent = main_menu
+
+    bodies = [
+        MBody(
+            title,
+            [
+                MBlk("*Default Messenger Content* 🏥1"),
+                MBlk("*Default Messenger Content* 2🏥"),
+            ],
+        ),
+        SBody(
+            title, [SBlk("*Default SMS Content* 1"), SBlk("*Default SMS Content*2 ")]
+        ),
+        UBody(
+            title,
+            [UBlk("*Default USSD Content* 1"), UBlk("*Default USSD Content* 2")],
+        ),
+        VBody(
+            title,
+            [VBlk("*Default Viber Content* 1"), VBlk("*Default Viber Content* 2")],
+        ),
+    ]
+    gender_variations = ["female", "male", "non-binary", "rather not say"]
+
+    for i in range(wa_body_count):
+        wa_body = f"*Default WhatsApp Content {i+1}* 🏥"
+
+        variation_messages = [
+            VarMsg(
+                wa_body.replace("Default", f"Varied {var+1}"),
+                gender=gender_variations[var],
+            )
+            for var in range(wa_var)
+        ]
+
+        bodies.append(
+            WABody(
+                title,
+                [
+                    WABlk(
+                        wa_body,
+                        variation_messages=variation_messages,
+                    )
+                ],
+            )
+        )
+
+    content_page = PageBuilder.build_cp(
+        parent=parent,
+        slug=title.replace(" ", "-"),
+        title=title,
+        bodies=bodies,
+        tags=[],
+        quick_replies=[],
+        triggers=[],
+        publish=True,
+    )
+    return content_page
+
+
+@pytest.mark.django_db
+class TestHasNextMessage:
+    # Test has_next_message
```

Comment:
This comment is redundant with the class name. If you feel the name is insufficient, rather add a docstring with more information.